composer.json

```
@@ -5,24 +5,24 @@
     "license": "MIT",
     "require": {
         "php": "^8.0",
-        "sylius/sylius": "^1.10",
+        "sylius/sylius": "~1.11.10 || ~1.12.0",
         "lexik/jwt-authentication-bundle": "^2.5",
-        "symfony/messenger": "^4.4|^5.2",
+        "symfony/messenger": "^5.2",
```

```suggestion
        "symfony/messenger": "^5.4",
```
and the same for the rest of SF components

---

https://github.com/Sylius/ShopApiPlugin/pull/738